clients/ts/signalr/src/WebSocketTransport.ts

```diff
@@ -44,7 +47,28 @@ export class WebSocketTransport implements ITransport {
 
         return new Promise<void>((resolve, reject) => {
             url = url.replace(/^http/, "ws");
-            const webSocket = new this.webSocketConstructor(url);
+            let webSocket: WebSocket | undefined;
+            let cookies: string = "";
+            if ((this.httpClient as any).getCookies) {
+                cookies = (this.httpClient as any).getCookies(url);
+            }
+
+            if (typeof window === "undefined") {
+                if (cookies) {
```

Comment:
roll this up into the previous condition `typeof window === "undefined" && cookies`